mysite/_config.php

```diff
@@ -12,20 +12,3 @@
 
 require_once('conf/ConfigureFromEnv.php');
 
-MySQLDatabase::set_connection_charset('utf8');
-
-// Set the current theme. More themes can be downloaded from
-// http://www.silverstripe.org/themes/
-SSViewer::set_theme('simple');
-
-// Disable BasicAuth by default
-BasicAuth::protect_entire_site(false);
-
-// Set the site locale
-i18n::set_locale('en_US');
-
-// Enable nested URLs for this site (e.g. page/sub-page/)
-if (class_exists('SiteTree')) SiteTree::enable_nested_urls();
-
-
-LeftAndMain::require_javascript('mysite/javascript/usabilla.js');
```

Comment:
This has disappeared - have you deleted the `usabilla.js` file from the project?